Various Errors when Installing Module

Apologies if this is me doing something incorrect, am new to golang.

I am running:
```
go version go1.15.5 linux/amd64
```

 When I run:

```
 import path does not begin with hostname
```

It returns with:
 ```
package citrix-netscaler-exporter/collector: unrecognized import path "citrix-netscaler-exporter/collector": import path does not begin with hostname
```

I came across https://stackoverflow.com/questions/47187280/unrecognized-import-path-import-path-does-not-begin-with-hostname

EDIT:

I was able to get around the above error by adding

```
export GO111MODULE=on
```

to my shell.

I am now seeing:

```
go get github.com/rokett/citrix-netscaler-exporter
go: downloading github.com/rokett/citrix-netscaler-exporter v0.0.0-20210223201402-f2112a388ff9
go: github.com/rokett/citrix-netscaler-exporter upgrade => v0.0.0-20210223201402-f2112a388ff9
go get: github.com/rokett/citrix-netscaler-exporter@v0.0.0-20210223201402-f2112a388ff9: parsing go.mod:
        module declares its path as: citrix-netscaler-exporter
                but was required as: github.com/rokett/citrix-netscaler-exporter
```

After coding around the above error I am now seeing:

```
go: github.com/gsperry2011/citrix-netscaler-exporter 4.5.2 => v0.0.0-20210308172404-e73d7091eb82
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308172404-e73d7091eb82/collector/aaa.go:6:2: package citrix-netscaler-exporter/netscaler is not in GOROOT (/usr/lib/golang/src/citrix-netscaler-exporter/netscaler)
```

Am now seeing:

```
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/connect.go:58:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/connect.go:67:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/disconnect.go:51:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/get_config.go:35:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/get_config.go:39:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/get_stats.go:35:14: undefined: io.ReadAll
../go/pkg/mod/github.com/gsperry2011/citrix-netscaler-exporter@v0.0.0-20210308173842-cd0377365b38/netscaler/get_stats.go:39:14: undefined: io.ReadAll
```

I'll keep updating as I go. Fixes here if anyone needs: https://github.com/gsperry2011/Citrix-NetScaler-Exporter/pull/1

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,9 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [4.5.2] - 2023-02-09
+## [4.6.0] - 2023-02-09
 ### Changed
 - #51 Ensure that idle HTTP client connections are closed after collecting metrics.
+- Corrected the import path to reflect the actual repository.
 
 ## [4.5.1] - 2020-09-14
 ### Fixed

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 FROM golang:alpine as builder
 
-ENV VERSION="4.5.2"
+ENV VERSION="4.6.0"
 
 WORKDIR $GOPATH/src/github.com/rokett
 RUN \

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 .PHONY: build
 
 APP = Citrix-NetScaler-Exporter
-VERSION = 4.5.2
+VERSION = 4.6.0
 BINARY-LINUX = ${APP}_${VERSION}_Linux_amd64
 
 BUILD_VER = $(shell git rev-parse HEAD)

--- a/collector/aaa.go
+++ b/collector/aaa.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/collector.go
+++ b/collector/collector.go
@@ -4,7 +4,7 @@ import (
 	"strconv"
 	"strings"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"

--- a/collector/cs_virtual_servers.go
+++ b/collector/cs_virtual_servers.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/gslb_services.go
+++ b/collector/gslb_services.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/gslb_virtual_servers.go
+++ b/collector/gslb_virtual_servers.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/interfaces.go
+++ b/collector/interfaces.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/service_groups.go
+++ b/collector/service_groups.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/services.go
+++ b/collector/services.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/virtual_servers.go
+++ b/collector/virtual_servers.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/collector/vpn_virtual_servers.go
+++ b/collector/vpn_virtual_servers.go
@@ -3,7 +3,7 @@ package collector
 import (
 	"strconv"
 
-	"citrix-netscaler-exporter/netscaler"
+	"github.com/rokett/citrix-netscaler-exporter/netscaler"
 
 	"github.com/prometheus/client_golang/prometheus"
 )

--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module citrix-netscaler-exporter
+module github.com/rokett/citrix-netscaler-exporter
 
 go 1.13
 

--- a/main.go
+++ b/main.go
@@ -14,7 +14,7 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 
-	"citrix-netscaler-exporter/collector"
+	"github.com/rokett/citrix-netscaler-exporter/collector"
 )
 
 var (

--- a/make.bat
+++ b/make.bat
@@ -2,7 +2,7 @@
 SETLOCAL
 
 set APP=Citrix-NetScaler-Exporter
-set VERSION=4.5.2
+set VERSION=4.6.0
 set BINARY-WINDOWS-X64=%APP%_%VERSION%_Windows_amd64.exe
 set BINARY-LINUX=%APP%_%VERSION%_Linux_amd64
 

